js/gated-content/src/components/event/EventTeaser.vue

```diff
@@ -68,22 +66,28 @@ export default {
   },
   computed: {
     date() {
-      return moment(this.video.attributes.date.value).format('YYYY-MM-DD');
+      return dayjs(this.video.attributes.date.value).format('YYYY-MM-DD');
     },
     time() {
-      return moment(this.video.attributes.date.value).format('h:mm a');
+      return dayjs(this.video.attributes.date.value).format('h:mm a');
     },
     duration() {
-      return moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment(this.video.attributes.date.end_value))).humanize();
+      const min = Math.floor(dayjs.duration(
+        dayjs(this.video.attributes.date.end_value) - dayjs(this.video.attributes.date.value),
+      ).asMinutes());
```

Comment:
https://day.js.org/docs/en/durations/minutes#docsNav
`As with the other getters for durations, dayjs.duration().minutes() gets the minutes (0 - 59).
dayjs.duration().asMinutes() gets the length of the duration in minutes.`
Please try to use
```suggestion
      ).minutes());
```

Reply:
.minutes() can not be used here